torch/_inductor/codegen/cuda/gemm_template.py

```diff
@@ -1395,7 +1418,7 @@ def _set_bias_layout_and_alignment(
     ) -> bool:
         import cutlass_library.library as cutlass_lib
 
-        has_bias = len(self.input_nodes) >= 3 and self.input_nodes[2] is not None
+        has_bias = len(self.input_nodes) == 3 and self.input_nodes[2] is not None
```

Comment:
intentional?

Reply:
ohh the nodes increased

Reply:
Yeah, the num nodes is now 5, I'm planning on adding bias support in another PR